src/dataexplorer/dataexplorer-application/src/components/PatientInfo/DisplayPatientInfo.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { IPatient } from 'types/IPatient';
+import { Row, Col, Card, Table } from 'antd';
+import './PatientInfo.style.scss';
+
+const DisplayPatientInfo = (props: {
+    patient: IPatient;
+    handleClick: any;
+    dataSource: fhir.ResourceBase[];
+}) => {
+    const name =
+        props.patient.name[0].given[0] + ' ' + props.patient.name[0].family;
+    const columns = [
+        {
+            title: 'Skjemanavn',
+            dataIndex: 'schemaName',
+            key: 'skjemanavn',
+        },
+        {
+            title: 'Innsendt',
+            dataIndex: 'submitted',
+            key: 'submitted',
+        },
```

Comment:
We should consistently use either norwegian or english as value for key, preferrably english imo.

Reply:
Think it's Norwegian because the title is shown in plain text in the view